src/App.vue

```diff
@@ -1,18 +1,20 @@
 <template>
-  <Hero />
-  <Hero :color="'#FF0000'" :height="50" />
-  <Hero :color="'#0000FF'" />
+  <Hero :height="43" />
+    <Input :label="'Email address'" />
+    <Input :label="'Password'" :typeInput="'password'" :placeHolder="'*********'"/>
+    <!-- <Form :fields="[{name: 'xname', type: 'text'}, {}]"/> -->
 </template>
-// <img alt="Vue logo" src="./assets/logo.png" />
```

Comment:
My suggestion is to take this comment Component outside `<template />` and set a TODO tag on it.

```suggestion
    /*
    TODO: Implement the following component after <Input /> is done.
    <Form :fields="[{name: 'xname', type: 'text'}, {}]"/> -->
    */
```